framework/src/dm/Make.defs

```diff
@@ -21,7 +21,7 @@ ifeq ($(CONFIG_LWM2M_CLIENT_MODE),y)
 CSRCS += dm_lwm2m.c
 endif
 CSRCS += dm_common_interface.c
-ifeq ($(CONFIG_NETUTILS_WIFI),y)
+ifeq ($(CONFIG_ARCH_CHIP_S5JT200),y)
```

Comment:
1. Here is framework, not arch. In my opinion, keeping architecture specific code in framework is not good.
2. When I see s5j_dm_connectivity.c, there is no s5j code. There are slsi_wifi codes. Is this right change?

Reply:
Those are very important points of view. Considering your comments, we will update this PR. Thanks.